index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
+  <link rel="stylesheet" href="fonts/css/all.css">
+  <title>Elena</title>
+</head>
+<body>
+    <div id="wrapper">
+      <div id="header">
+        <div class="container">
+          <div class="logo">
+            <a href="#"><img src="img/logo.png" alt="Logo"></a>
+          </div>
+          <div class="nav">
+            <input type="checkbox" name="toggle" id="menu" class="toggleMenu">
+            <label for="menu" class="toggleMenu"><i class="fa fa-bars"></i>Menu</label>
+            <ul>
+              <li><a href="#">HOME</a></li>
```

Comment:
It's preferable to use `text-transform: uppercase;` to make all the word uppercase.